seclang/directives_test.go

```diff
@@ -181,3 +181,19 @@ func TestInvalidRulesWithIgnoredErrors(t *testing.T) {
 		t.Error("failed to error on invalid rule")
 	}
 }
+
+func TestSecDataset(t *testing.T) {
```

Comment:
We need to include more tests here. I guess the char "`" isn't supported in the dataset.

Reply:
"`" is supported in the dataset, the code only uses interprets it if it is the last character of the directive declaration or the only character in the line.